package/scripts/lib.js

```diff
@@ -114,6 +114,8 @@ export function rollupAMDOutput(type) {
         dir: path.join(packageFolder, 'release', type),
         sourcemap: !process.env.CI,
         plugins: [type === 'min' && terser()],
-        globals: Object.fromEntries(languageServiceFiles),
```

Comment:
Unused. (Accidentally left over from when I was still trying to get the build system working)